app/routes/public/cfs.js

```diff
@@ -1,15 +1,21 @@
 import Ember from 'ember';
+import moment from 'moment';
 
-const { Route, RSVP } = Ember;
+const { Route, RSVP, computed } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Call for Speakers');
   },
 
   model() {
+    const eventDetails = this.modelFor('public');
     return RSVP.hash({
-      event: this.modelFor('public')
+      event        : eventDetails,
+      speakersCall : eventDetails.get('speakersCall'),
+      isOpen       : computed('eventDetails.endsAt', 'speakersCall.endsAt', function() {
+        return moment(this.get('eventDetails.endsAt')).isAfter(this.get('speakersCall.endsAt'), 'year');
```

Comment:
@niranjan94 
What I wanted to write was that if event's end date is after speakerCall start date then it will be opened else closed.
I meant to write this
```moment(this.get('eventDetails.endsAt')).isAfter(this.get('speakersCall.startsAt'), 'year')```